### DISCUSSION TITLE
Another postcss-color-hex-alpha?

### DISCUSSION THREAD

**Discussion by yoyo837** [csstools/postcss-plugins#759] [General]
When I delete `postcss-color-hex-alpha` from `node_modules`, the css build is still successful, and all `rgba` is successfully converted to `rrggbbaa`, who else does this except `postcss-color-hex-alpha`?

**Comment by romainmenke** [comment]
🤔 That is impossible to answer.

What I would do :
1. remove all postcss plugins
2. add them back one by one
3. each time `rgba` is converted to `rrggbbaa`
  3.1. add a comment next to that plugin in your config : "converts `rrggbbaa`"
  3.2. remove that plugin again
4. go to `2`

----

I would guess that a minifier like CSSNano might also convert these values.

**Reply by yoyo837** [reply]
Thanks. I thought about this method, but I can only delete it one by one from `node_modules`, because I am using `@vue/cli` and have not configured any `postcss` plugins for it.

**Reply by romainmenke** [reply]
Do you have a link to docs for `@vue/cli` and `postcss`?
I have never used anything related to `vue` :)

**Reply by yoyo837** [reply]
https://cli.vuejs.org/guide/css.html#postcss
Documentation doesn't provide much useful information.
The source code here might be more helpful:
https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/config/css.js

**Reply by romainmenke** [reply]
It does have cssnano : https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/package.json#L49

Weird that there aren't clear docs on how to take over full control of PostCSS.

**Reply by yoyo837** [reply]
Sadly. Can not seem to do that by `@vue/cli`.

**Reply by yoyo837** [reply]
Thanks again.

**Comment by yoyo837** [comment]
> I would guess that a minifier like CSSNano might also convert these values.

Let me debug it.

**Reply by yoyo837** [reply]
Sorry forgot to reply, it is indeed converted by cssnano in @vue/cli.
